tailwind.config.js

```diff
@@ -17,6 +17,8 @@ module.exports = {
     },
     extend: {
       colors: {
+        datatableborder: '#27272A',
+        datatableheader: '#A1A1AA',
```

Comment:
These are not necessary. 

`#27272A` is equal to Tailwind's `zinc-800` which is the same as `border-border` in our project. 

`#A1A1AA` is equal to tailwind's `zinc-400` which is the same as 

Please delete this and replace it in the component(s) with `zinc-400` wherever used.

Let me know if you have any questions.